src/__tests__/optimize.test.ts

```diff
@@ -505,3 +480,141 @@ describe('Optimize 4 parameters of a linear combination of gaussian and lorentzi
     }
   });
 });
+
+describe('Sum of a mix of distributions', () => {
+  it('group of two GL', () => {
+    let peaks = [
+      {
+        x: 0,
+        y: 0.001,
+        fwhm: 0.31,
+        shape: {
+          kind: 'pseudoVoigt',
+          options: { mu: (xFactor * nbPoints) / 10 },
+        } as Shape1D,
+      },
+      {
+        x: 0,
+        y: 0.001,
+        fwhm: 0.31,
+        shape: {
+          kind: 'pseudoVoigt',
+          options: { mu: (xFactor * nbPoints) / 10 },
+        } as Shape1D,
+      },
+      { x: -0.5, y: 0.001, fwhm: 0.31, shape: { kind: 'gaussian' } },
+      { x: 0.5, y: 0.001, fwhm: 0.31, shape: { kind: 'gaussian' } },
+      { x: -0.5, y: 0.001, fwhm: 0.31, shape: { kind: 'lorentzian' } },
+      { x: 0.5, y: 0.001, fwhm: 0.31, shape: { kind: 'lorentzian' } },
+    ];
+
+    let pTrue = [
```

Comment:
Why does this array exists ? Just make the tests with the values that are defined in the peaks it would be more readable.